app.config.ts

```diff
@@ -17,6 +17,7 @@ export default defineAppConfig({
       { text: 'Node Providers', href: '/node-providers' },
       { text: 'Blog', href: '/blog' },
       { text: 'Docs', href: 'https://docs.storacha.network' },
+      { text: 'Referrals', href: 'https://storacha.network/referrals' },
```

Comment:
```suggestion
      { text: 'Referrals', href: '/referrals' },
```
internal app pages should be relative